### ISSUE TITLE
Segmentation Fault on interleaved fq files

### ISSUE THREAD

**Issue by millerh1** [OpenGene/fastp#382]
Thank you for making this very useful tool! 

I am experiencing an error when using interleaved fastq files:

```shell
$ fastp -i fastqs_interleave/S1IP_1_val_hg38__interleave.fastq --interleaved_in
Segmentation fault (core dumped)
```

This error only happens in v0.23.0 and v0.23.1 -- downgrading to v0.22.0 fixes the issue.

**Comment by sfchen** [comment]
I will fix this soon.

**Comment by sfchen** [comment]
fixed, please update to v0.23.2

**Comment by millerh1** [comment]
Awesome -- just tested it and it works fine! Thank you! 😎
